src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_daemon_thread.py

```diff
@@ -26,6 +26,11 @@ def __init__(self, py_db, target_and_args=None):
         mark_as_pydevd_daemon_thread(self)
         self._target_and_args = target_and_args
 
+        if hasattr(self, 'name'):
```

Comment:
Is this really needed? Ideally I think we could just set `thread.name = name` where needed.